app/src/main/java/org/simple/clinic/medicalhistory/sync/MedicalHistoryPayload.kt

```diff
@@ -43,6 +43,12 @@ data class MedicalHistoryPayload(
     @Json(name = "hypertension")
     val hasHypertension: Answer?,
 
+    @Json(name = "smoking")
+    val isSmoker: Answer,
```

Comment:
Maybe we can name it as `isSmoking` or `smoking` to match the API naming convention? Same for database column naming as well.

Reply:
Renamed to `smoking`